ui/app/components/signature-request.js

```diff
@@ -54,7 +55,7 @@ SignatureRequest.prototype.renderHeader = function () {
 
     h('div.request-signature__header-background'),
 
-    h('div.request-signature__header__text', 'Signature Request'),
+    h('div.request-signature__header__text', t('sigRequest')),
```

Comment:
I don't see `sigRequest` in `app/_locales/en`. I see `sigRequested` and `yourSigRequested`. @mapmeld Was `sigRequested` the preferred one?

Reply:
It looks like there are actually three different messages used ("Signature Request", "Signature Requested", "Your Signature Requested").  I would recommend either adding a ```sigRequest``` entry to the locales file, or changing the line here (and message) to ```t('sigRequested')```

Reply:
Ok thanks for the feedback. Also, is it a typo `sigRequest` vs `signRequest`?

Reply:
No I don't think so... my guess is when I alphabetized the messages, I didn't notice that "Signature Request" and "Signature Requested" were different.
If you want the UI to be exactly the same, add ```sigRequest``` as an entry in the locales file.
If you want to change this one line, it has to be a code already present in the locales file (```sigRequested``` would work if you feel the message has the same meaning)